integration_test/cases/idle_test.exs

```diff
@@ -11,7 +11,7 @@ defmodule TestIdle do
     execute_test_case(agent, opts)
   end
 
-  @tag :idle_timeout
+  @tag :idle_hibernate
```

Comment:
We should support this for all pools that can do `idle_timeout`, I left a comment on how to achieve this previously.

Reply:
Actually lets keep this for now.